.github/workflows/ci_pr.yaml

```
@@ -0,0 +1,97 @@
+      - id: set_max_parallel
+        name: set_max_parallel
+        run: |
+          echo "max_parallel=${{ env.max_parallel }}" >> $GITHUB_OUTPUT
+
+  run_system_tests:
+    needs: populate_env_vars
+    name: "Run system tests"
+    uses: LightBitsLabs/lbcitests/.github/workflows/ci-base.yaml@main
+    with:
+      manifest_name: ${{ needs.populate_env_vars.outputs.manifest_name }}
+      manifest_branch: ${{ inputs.manifest_branch }}
```

actionlint

property "manifest_branch" is not defined in object type {manifest_name: string}

(expression)